src/utils/docx/writeScenariosTable.js

```diff
@@ -0,0 +1,188 @@
+import {
+  Paragraph,
+  Table,
+  TableRow,
+  WidthType,
+  TableCell,
+  ShadingType,
+} from 'docx';
+import { list, info } from '@geops/tree-lib';
+import { getAZ, getScenarios, getScenarioColumns } from '../projectionUtils';
+import { cellPadding, treeTypesReducer } from './utils';
+
+const getScenarioColumn = (scenario, projection, language, t) => {
+  const { forestType, transitionForestType } = projection;
+  const altitudinalZone = getAZ(projection.altitudinalZone);
+  const scenarios = getScenarios(scenario, t);
+  const treeValues = list(projection, true).slice(0, 3);
+
+  return (
+    forestType && {
+      header: `${
+        transitionForestType
+          ? `${forestType} (${transitionForestType}) `
+          : `${forestType} `
+      } ${info('altitudinalZone', altitudinalZone)[language]}`,
+      subHeader: scenarios.names.join(', '),
+      dominantType: info('treeType', treeValues[0][0]),
+      importantTypes: treeValues[1].map((code) => info('treeType', code)),
+      otherTypes: treeValues[2].map((code) => info('treeType', code)),
+    }
+  );
+};
+
+export const writeScenariosTable = (
+  location,
+  projectionResult,
+  projectionMode,
+  latinActive,
+  language,
+  t,
+) => {
+  const columns = getScenarioColumns(
+    location,
+    projectionMode,
+    projectionResult,
+    getScenarioColumn,
+    language,
+    t,
+  );
+  const backgroundColorHeader = {
+    fill: 'bbe0e3',
+    type: ShadingType.CLEAR,
+    color: 'auto',
+  };
+  const backgroundColorCell = {
+    fill: 'e7f3f4',
+    type: ShadingType.CLEAR,
+    color: 'auto',
+  };
+  return new Table({
```

Comment:
Does it make sense to extract some helper function(s) for table creation?

Reply:
Done. I removed a lot of code duplicates and wrote some helper methods for the cell creation